src/NativeScript/TNSRuntime.mm

```diff
@@ -36,6 +44,15 @@ - (instancetype)initWithApplicationPath:(NSString*)applicationPath {
         self->_applicationPath = [applicationPath copy];
         WTF::wtfThreadData().m_apiData = static_cast<void*>(self);
 
+#if PLATFORM(IOS)
+        VM* vm = self->_vm.get();
+        self->_memoryPressureNotificationSubscription = [[NSNotificationCenter defaultCenter] addObserverForName:UIApplicationDidReceiveMemoryWarningNotification object:nil queue:nil usingBlock:^(NSNotification *note) {
+            JSLockHolder lock(vm);
+            vm->heap.collect(FullCollection);
+            vm->heap.releaseDelayedReleasedObjects();
```

Comment:
Is this needed? It seems to me that `releaseDelayedReleasedObjects` will be called after the lock is released.